### ISSUE TITLE
RaisedButton deprecated

### ISSUE THREAD

**Issue by alex-lechner** [escamoteur/rx_widgets#23]
Hey!

I've updated flutter to version `3.1.0` (from the flutter master channel) and tried to start my application but received the following error due to `RaisedButton` being deprecated now:

```sh
../../flutter/.pub-cache/hosted/pub.dartlang.org/rx_widgets-4.0.1/lib/src/rx_raised_button.dart:55:16: Error: The method 'RaisedButton' isn't defined for the class 'RxRaisedButton'.
 - 'RxRaisedButton' is from 'package:rx_widgets/src/rx_raised_button.dart' ('../../flutter/.pub-cache/hosted/pub.dartlang.org/rx_widgets-4.0.1/lib/src/rx_raised_button.dart').
Try correcting the name to the name of an existing method, or defining a method named 'RaisedButton'.
        return RaisedButton(
               ^^^^^^^^^^^^
```

I'm using rx_widgets version `4.0.1`

Here's my dart and flutter version:
```sh
Flutter 3.1.0-0.0.pre.1910 • channel master • https://github.com/flutter/flutter.git
Framework • revision f82178e2d2 (5 hours ago) • 2022-07-27 10:22:07 -0700
Engine • revision 328a4b24e2
Tools • Dart 2.19.0 (build 2.19.0-39.0.dev) • DevTools 2.15.0
```

**Comment by Titos23** [comment]
Hello. In fact ```RaisedButton``` is deprecated. Instead use ````ElevatedButton````.

**Comment by escamoteur** [comment]
Is anyone interested in taking over this package?

**Comment by AlexBacich** [comment]
@escamoteur , I am. I've already contributed in the past.

**Comment by AlexBacich** [comment]
Done, I'm now owner and issue is resolved with 4.1.0.
